### ISSUE TITLE
UBER: user feedback on 3.0 new UI

### ISSUE THREAD

**Issue by elrayle** [cul-it/exhibits-library-cornell-edu#490]
Craig Mains provided the following feedback on the UI for the Spotlight 3.0 upgrade

#### Title/Subtitle/Headers related feedback
* All are described in Issue #542
* ONLINE EXHIBITIONS grey banner adds clutter and competes with title of exhibit. I believe the previous header looked better. 
Exhibit Main Titles in all caps are difficult to read.  
* Main exhibit title and subtitle could be closer together and dropped lower to the shaded area of the banner. 
* The secondary title text is now easier to read as it is all white. Maybe switch style to all white main title and drop shadow subtitle smaller font. 
* General header titles increased in size. Headers are larger than exhibit title and seem too big for the page. They seem too big on cell phone view as well. 

#### Additional UI related feedback
* Image record pages loading awkward composition.  Image size increase is better, but text should line up to the left of image or directly below with image more centered. (if +/- controls were at the bottom, the image and text could be justified at the top. )  Issue #544
* When selecting an image, the Image record page has Back to Search when no search was initiated.  Issue #543
* The widget providing button "highlighting individual pages" in the exhibit has decreased the size of the image and increased the font. The previous proportions worked better.  Same with the launch page for all exhibits.  Button images are smaller making the feature image smaller and title text now covers much of the image.  I’m thinking websites about exhibits should highlight the images in the exhibit more not less.  Issues #546, #545

### Editing issues:  

* The widget that highlights pages from the exhibit doesn’t bring up titles when you type them in and doesn’t allow you to enter titles into Enter a title.. Field.  Issue #503
* Image uploads for horizontal rows are limited to 1meg but load page doesn’t say there’s a limit.  (I like the added link option here!)  Issue #504
* Some of the item links in my unpublished “Bluest Eye” exhibit were broken.  Images on Lorain County, three in a row.  (would be nice if you could sort by title when going through exhibit items) . Site gave me an error while trying to look through my items here. Issue #505

**Comment by mhk33** [comment]
Some responses to some of the General UI feedback:

- ONLINE EXHIBITIONS grey banner adds clutter and competes with title of exhibit. I believe the previous header looked better. --> _This is the direction we have received and has been approved by Library Communications and Central Branding, we likely need to work with what we have here._
- Exhibit Main Titles in all caps are difficult to read. --> _Agreed, will change and bump up font size_
- Main exhibit title and subtitle could be closer together and dropped lower to the shaded area of the banner. --> _Will review the font size and positioning._
- The secondary title text is now easier to read as it is all white. Maybe switch style to all white main title and drop shadow subtitle smaller font. --> _Will review font size and positioning in relation to each other._
- Image record pages loading awkward composition. Image size increase is better, but text should line up to the left of image or directly below with image more centered. (if +/- controls were at the bottom, the image and text could be justified at the top. ) --> _Will review layout for this page._
- General header titles increased in size. Headers are larger than exhibit title and seem too big for the page. They seem too big on cell phone view as well. --> _Will take a look._
- When selecting an image, the Image record page has Back to Search when no search was initiated. --> _Needs closer investigation. Not sure if this is in an overridden template or an issue with core app._

**Comment by mhk33** [comment]
I just merged into dev these updates:

- Font size and style for the exhibit title, subtitle and navigation
- Added space between the metadata and openseadragon viewer on the item page.

Regarding this comment - "Image record pages loading awkward composition. Image size increase is better, but text should line up to the left of image or directly below with image more centered." --> Going to leave it as is, as I didn't want to get into too many template and layout overrides, at least not at the moment. I think we could consider this in the future.

Also, the overall heading sizes could be improved in relation to the site title and H1. I think it's a bit better now with the changes I just pushed, but could be improved later as well.
